tasks/create_http_task.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START cloud_tasks_create_http_task]
+
+// Command create_task constructs and adds a task to an App Engine Queue.
+package main
+
+import (
+	"context"
+	"fmt"
+
+	cloudtasks "cloud.google.com/go/cloudtasks/apiv2beta3"
+	taskspb "google.golang.org/genproto/googleapis/cloud/tasks/v2beta3"
+)
+
+// createTask creates a new task in your App Engine queue.
+func createHTTPTask(projectID, locationID, queueID, url, message string) (*taskspb.Task, error) {
```

Comment:
Can we move any of these arguments to be variables in the function? With the new canonical sample format, we're trying to keep args to a minimum. For example,

```
	locationID := "us-central1"
	queueID := "my-queue"
	url := "https://example.com/task_handler"
```

Reply:
This is following the pattern of the existing snippet. Can refactoring the basic pattern be done in a follow-up?

Reply:
SGTM.